paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -619,9 +620,8 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
       bool is_conv3d = strides.size() == 3U;
 
       PADDLE_ENFORCE_NE(is_conv3d, true,
-                        platform::errors::InvalidArgument(
-                            "int8 does not support conv3d currently, should "
-                            "set param is_conv3d as False"));
```

Comment:
This error should have a type？

Reply:
I changed the error type to platform::errors::Unimplemented because int8 conv3d is not supported yet.